src/olympia/constants/activity.py

```diff
@@ -814,20 +814,20 @@ class VERSION_RESIGNED(_LOG):
 class FORCE_DISABLE(_LOG):
     id = 167
     keep = True
-    # We don't want to notify developers, this is not a regular rejection - the
-    # add-on is likely malicious.
-    hide_developer = True
     reviewer_review_action = True
-    format = '{addon} force-disabled by {user_responsible}.'
+    format = '{addon} force-disabled'
```

Comment:
`format` now needs to be translated for both of these actions (not needed for `reviewer_format` nor `admin_format`)